src/Parser/FeralDruid/Modules/Talents/SavageRoarDmg.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber , formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+import getDamageBonus from '../FeralCore/getDamageBonus';
+
+const SAVAGE_ROAR_DAMAGE_BONUS = .15;
+
+class SavageRoar extends Module {
+  static dependencies = {
+    enemies: Enemies,
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+
+  on_initialized() {
+    this.active = this.combatants.selected.hasTalent(SPELLS.SAVAGE_ROAR_TALENT.id);
+  }
+
+  on_damage(event) {
+    if (this.combatants.selected.hasBuff(SPELLS.SAVAGE_ROAR_TALENT.id, event.timestamp) && event.targetIsFriendly === false) {
```

Comment:
1. It's generally cleaner to move `!event.targetIsFriendly` above this if-statement and do an early `return`; grouping unrelated if-statements makes it harder to see.
2. Is this supposed to not be filtered byPlayer (`on_byPlayer_damage`)?

Reply:
Changed the targetIsFriendly check to a separate statement and changed to on_byPlayer_damage